webapp/package.json

```diff
@@ -16,7 +16,7 @@
   "author": "",
   "license": "",
   "devDependencies": {
-    "@babel/cli": "7.5.5",
+    "@babel/cli": "^7.8.4",
```

Comment:
Can you save these as exact versions by removing the `^` from the version numbers?

Reply:
`npm install` may need to be run afterwards to update the lock file.

Reply:
Is there an automated way to do this?  I've only seen the `npm audit fix` solution used.

Reply:
I'm not sure. You may need to do it manually